src/main/kotlin/calculator/Calculator.kt

```diff
@@ -1,58 +1,26 @@
 package calculator
 
-import calculator.exceptions.NumberSeparationException
 import calculator.exceptions.OperationBetweenNumbersException
 
 object Calculator {
     fun calculate(input: String?) = calculate(InputParser.validateInput(input))
```

Comment:
I prefer declaring types for functions and constants because it makes the code easier to understand at a glance. It also helps prevent accidentally changing the return type when the implementation changes. I hope the following item from Effective Kotlin is helpful:
– Item 13: Consider making types explicit